### ISSUE TITLE
anchor_size

### ISSUE THREAD

**Issue by FuyuWang** [balancap/SSD-Tensorflow#83]
Hello! how do you compute anchor_sizes in ssd_vgg_300.py? When I user formula in paper, it is not consistent with your's.

**Comment by whigy** [comment]
I have the same question! Have you solved it?

**Comment by FuyuWang** [comment]
Well, it is implemented in function "ssd_size_bounds_to_values" in file "ssd_vgg_300.py" , but it is not the very same as paper.

**Comment by whigy** [comment]
Yes, I also do not think it aligns with the paper. However, is this function really be called somewhere? I found the anchor sizes already defined as:
`anchor_sizes=[(21., 45.),
                      (45., 99.),
                      (99., 153.),
                      (153., 207.),
                      (207., 261.),
                      (261., 315.)],`
And when building the network just directly uses the predifined sizes.

**Comment by FuyuWang** [comment]
If you call this function, you can get anchor_sizes=[(21., 45.), (45., 99.), (99., 153.), (153., 207.), (207., 261.), (261., 315.)].  I think the author used the results directly without calling the function.

**Comment by nlngh** [comment]
Have you guys saw any difference in performance if you use authors implementation or the as in paper?

**Comment by xbcReal** [comment]
@Erwin-fy As you said,what is the meaning of the params of the func ssd_size_bounds_to_values,ie,size_bounds,n_feat_layers?
